### DISCUSSION TITLE
有字幕情况下显示字幕列表在视频侧方

### DISCUSSION THREAD

**Discussion by Benature** [the1812/Bilibili-Evolved#3867] [功能建议 / Ideas]
想要和 Youtube 的「显示解说词」一样的效果 🤗

**Comment by the1812** [comment]
细说? 我没怎么用过 Youtube

**Reply by Benature** [reply]
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/35028647/207367853-53036a5b-0f2c-4d26-9bd6-45946613848b.png">

大概如上图所示，列表形式展现字幕内容

**Reply by Benature** [reply]
更近一步的话就是随着播放进度自动滚动这个列表，但是可能实现比较繁琐。不知道能不能先有这个列表展示出来呢？

**Reply by the1812** [reply]
挺麻烦的, 等个有缘人写吧, 我是没啥兴趣搞这个(

**Reply by Benature** [reply]
哈哈好

**Comment by WakelessSloth56** [comment]
正好有类似的需求，就简单写了一下。
大致基本功能实现了，跟随播放自动滚动没有做，确实比较繁琐。
样式也还需要优化。

预览：
![20230327012510](https://user-images.githubusercontent.com/43951496/227793616-a66ad9f1-817f-45b6-aa40-4aa0bca9165d.gif)

**Reply by WakelessSloth56** [reply]
目前各方面都比较粗糙，暂时就先不发PR了（

**Reply by WakelessSloth56** [reply]
做了一下自动滚动的功能。

![20230327233012](https://user-images.githubusercontent.com/43951496/227989044-0b20ba1b-47cc-4340-8dbb-e878b8745bdd.gif)

不过下面的评论已经有更好的东西了，我再考虑考虑要不要把它做完（

**Comment by Ethkuil** [comment]
可以尝试 Chrome扩展 [哔哩哔哩字幕列表](https://chrome.google.com/webstore/detail/%E5%93%94%E5%93%A9%E5%93%94%E5%93%A9%E5%AD%97%E5%B9%95%E5%88%97%E8%A1%A8/bciglihaegkdhoogebcdblfhppoilclp)。列表、滚动都有，还有把所有字幕合并在一起的文章视图。填写OpenAI的API后还可以调用OpenAI的模型自动翻译、总结。

除了要多装一个扩展外没有任何缺点，完美符合需求。我一直在用，感觉很好。

**Reply by WakelessSloth56** [reply]
好嘛，我又在造轮子了。 （我才刚把滚动写完 :joy:
不过好像它这个好像不开源？
